.github/workflows/wasm.yml

```diff
@@ -0,0 +1,113 @@
+name: Build Wasm Modules
+on: [ push, pull_request ]
+
+env:
+  SERENITY_SOURCE_DIR: ${{ github.workspace }}
+
+concurrency: wasm
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    if: github.repository == 'SerenityOS/serenity'
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+      - name: "Install Ubuntu dependencies"
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ninja-build gcc-12 g++-12 libstdc++-12-dev
+      - name: "Install emscripten"
+        uses: mymindstorm/setup-emsdk@v12
+        with:
+          version: 3.1.25
```

Comment:
Is this the same version we used to have, or latest stable?

Reply:
it's exactly what we had before the workflow was removed.

Reply:
Worth an update? or is that for "later :tm: " :)

Reply:
"later :tm:" sounds fine to me :P